### PR TITLE
Fix radarr to rescan and run rename commands

### PR DESCRIPTION
**Pull request by ant385525**
## CLA

[X] I agree that by opening a pull requests I am handing over copyright ownership of my work contained in that pull request to the FileFlows project and the project owner. My contribution will become licensed under the same license as the overall project.

### CODE REVIEW

File: Scripts/Flow/Applications/Radarr/Radarr - Rename.js [hunk 1]
```diff
@@ -1,23 +1,135 @@
 import { Radarr } from 'Shared/Radarr';
 /**
  * This script will send a rename command to Radarr
- * @author Shaun Agius
+ * @author Anthony Clerici
  * @version 1.0.0
- * @revision 4
+ * @revision 1
```
Comment (revenz) [inline Scripts/Flow/Applications/Radarr/Radarr - Rename.js:6]: cant go down in revisions, also why change the authors name?

**Review by revenz — CHANGES_REQUESTED** [review]

File: Scripts/Flow/Applications/Radarr/Radarr - Rename.js (hunk 1 above)
Comment (ant385525) [reply Scripts/Flow/Applications/Radarr/Radarr - Rename.js:6]: It's completely rewritten, the other script didn't do anything so I was considering it a new script. Should the revision still go up? and I can add the other author's name back, I don't code much so dk the best practices

Comment (revenz) [reply Scripts/Flow/Applications/Radarr/Radarr - Rename.js:6]: the revision still needs to be higher so it actually gets updated.  the revision is how it determines if there is a newer version.

Have you spoken to the original author, are they happy with replacing this?

Comment (ant385525) [reply Scripts/Flow/Applications/Radarr/Radarr - Rename.js:6]: I see ok. And yeah see the discord in channel scripts-code-functions. The other version wasn't functional

**Pull request merged**